rules/klayout/drc/testing/run_switch_checking.py

```diff
@@ -97,17 +93,15 @@ def check_output_log(log_file, expected_result):
     expected_log = df["expected_logs"]
 
     ## init log file
-    os.system('mkdir -p {0}'.format(RUN_DIRECTORY))
-    os.system('touch {0}temp.log'.format(RUN_DIRECTORY))
-    os.system('mv -f {0}temp.log {0}{1}'.format(RUN_DIRECTORY,LOG_FILE_NAME))
+    subprocess.run(['mkdir', '-p', '{0}'.format(RUN_DIRECTORY)])
```

Comment:
These should all be `shutil` commands.

Reply:
Actually, 'Path' from 'pathlib'. I recommend to only use shutil when pathlih does not provide the required functionality.